spec/class.dd

```diff
@@ -1009,7 +1006,6 @@ $(H2 $(LNAME2 allocators, Class Allocators))
 $(B Note): Class allocators are deprecated in D2.
 $(GRAMMAR
 $(GNAME Allocator):
-    $(D new) $(GLINK2 function, Parameters) $(D ;)
```

Comment:
Even though class allocators are obsolete it seems that the parser does not complain about the following:

```d
class A
{
    new(size_t size);    
    
}
```

Reply:
This deletion comes from this commit:
```
spec: Remove definitions redundant with MissingFunctionBody

$(D ;) is already a possible FunctionBody (as MissingFunctionBody).
```